cli/command/formatter/sorter.go

```diff
@@ -0,0 +1,319 @@
+package formatter
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+)
+
+const (
+	ascendantKey  = "asc"
+	descendantKey = "dsc"
```

Comment:
I can't be sure, but looks like you are right:

https://www.allacronyms.com/descendant/abbreviated
http://www.rootsweb.ancestry.com/~rigenweb/abbrev.html

I'll modify it to desc